qCC/ccCommandLineCommands.cpp

```diff
@@ -6292,23 +6295,27 @@ bool CommandICP::process(ccCommandLineInterface& cmd)
 				QString rotation = cmd.arguments().takeFirst().toUpper();
 				if (rotation == "XYZ")
 				{
-					transformationFilters = CCCoreLib::RegistrationTools::SKIP_NONE;
+					//keep this to not make breaking changes
 				}
 				else if (rotation == "X")
 				{
-					transformationFilters = CCCoreLib::RegistrationTools::SKIP_RYZ;
+					transformationFilters |= CCCoreLib::RegistrationTools::SKIP_RYZ;
```

Comment:
why the "|' operator? Did you imagine a use case where -ROT would be used multiple times for a single ICP command? It won't be considered by the registration algorithm anyway... The algorithm only considers one SKIP_R?? option at a time.

Reply:
you are right, but what if someone send a -SKIP_TX before -ROT?

Reply:
Ah ok, that's for the SKIP_T?, my bad...